modules/Cockpit/assets/components/field-select.tag

```diff
@@ -2,7 +2,7 @@
 
     <select ref="input" class="uk-width-1-1 {opts.cls}" bind="{ opts.bind }">
         <option value=""></option>
-        <option each="{ option,idx in options }" value="{ option }">{ option }</option>
+        <option each="{ option,idx in options }" value="{ option }" selected="{ parent.root.$value === option }">{ option }</option>
```

Comment:
is _selected_ needed here as `this.refs.input.value = this.root.$value;` does that? 🤔